src/Events/Authorization/XacmlMapperHelper.cs

```diff
@@ -17,15 +17,17 @@ public static class XacmlMapperHelper
     private const string UserPrefix = "/user/";
     private const string OrgPrefix = "/org/";
     private const string PartyPrefix = "/party/";
-    private const string OrganizationPrefix = "/organization/";
+    private const string OrganisationPrefix = "/organisation/";
```

Comment:
shouldn't this also be 'z'  ? ref comment on line 27.

Reply:
The comment is for a specific value used by the Authorization logic. I would have used `s` if I could.
I'll try to make it more obvious what I ment.